src/codec_libgav1.c

```diff
@@ -33,28 +33,24 @@ static avifBool gav1CodecOpen(avifCodec * codec)
     return AVIF_TRUE;
 }
 
-static avifBool gav1CodecGetNextImage(struct avifCodec * codec, avifDecodeSample * sample, avifBool alpha, avifImage * image)
+static avifBool gav1CodecGetNextImage(struct avifCodec * codec, const avifDecodeSample * sample, avifBool alpha, avifImage * image)
 {
+    if (Libgav1DecoderEnqueueFrame(codec->internal->gav1Decoder,
+                                   sample->data.data,
+                                   sample->data.size,
+                                   /*user_private_data=*/0,
+                                   /*buffer_private_data=*/NULL) != kLibgav1StatusOk) {
+        return AVIF_FALSE;
+    }
+    // Each Libgav1DecoderDequeueFrame() call invalidates the output frame
+    // returned by the previous Libgav1DecoderDequeueFrame() call. Clear
+    // our pointer to the previous output frame.
+    codec->internal->gav1Image = NULL;
     const Libgav1DecoderBuffer * nextFrame = NULL;
-    // Check if there are more samples to feed
-    if (sample) {
```

Comment:
It is not easy to see the changes in this diff view. There are only two changes here:

1. I removed the if (sample) check because we can assume 'sample' is not NULL.
2. I moved the declaration of nextFrame (at line 38) to be right before the Libgav1DecoderDequeueFrame() call.

Reply:
Argh, there is a third change: I removed the "Check if there are more samples to feed" and "Feed another sample" comments.